db/schema.rb

```diff
@@ -11,20 +11,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160701215002) do
```

Comment:
I rebased out most of these changes, leaving just the two table definitions. Is it good practice to avoid updating the schema version as well?